homeassistant/components/sleepiq/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We need a `strings.json` that is used to generate the translations

Run `python3 -m script.translations develop` to regenerate it.

Please use key common as much as possible in `strings.json`